au4/src/trackedit/itrackeditinteraction.h

```diff
@@ -26,6 +27,11 @@ class ITrackeditInteraction : MODULE_EXPORT_INTERFACE
     virtual muse::async::Channel<ClipKey, double /*newStartTime*/, bool /*completed*/> clipStartTimeChanged() const = 0;
 
     virtual bool changeClipTitle(const ClipKey& clipKey, const muse::String& newTitle) = 0;
+    virtual void clearClipboard() = 0;
+    virtual bool paste(double begin, TrackId trackId) = 0;
```

Comment:
Won't harm to mention the clipboard in the names of the added functions

Reply:
fixed